notebooks/env.ipynb

```diff
@@ -0,0 +1,628 @@
+{
```

Comment:
<p>This explanation should be moved up to where the gym stuff is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/chandar-lab/RLHive/pull/305/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='31'/>